users/README.md

```diff
@@ -265,3 +266,36 @@ Updates data of the User.
   - **Code:** 503
     - **Content:**
       `{ 'statusCode': 503, 'error': 'Service Unavailable', 'message': 'Something went wrong please contact admin' }`
+
+## **PATCH /users/:id/roles**
+
+Updates roles for the User.
+
+- **Params**  
+  None
+- **Query**  
+  None
+- **Headers**  
+  Content-Type: application/json
+- **Cookie**  
+  rds-session: `<JWT>`
+- **Body**
+  `{
+    rolename: <boolean>
+  }`
+- **Success Response:**
+  - **Code:** 204
+    - **Content:** `{ 'message': 'role updated successfully!'}`
+- **Error Response:**
+  - **Code:** 404
+    - **Content:**
+      `{ 'statusCode': 404, 'error': 'Not Found', 'message': 'User not found' }`
+  - **Code:** 400
+    - **Content:**
+      `{ 'statusCode': 400, 'error': 'Invalid Request', 'message': 'Invalid role' }`
+  - **Code:** 401
+    - **Content:**
+      `{ 'statusCode': 401, 'error': 'Unauthorized', 'message': 'Unauthenticated User' }`
+  - **Code:** 503
+    - **Content:**
+      `{ 'statusCode': 503, 'error': 'Service Unavailable', 'message': 'Something went wrong please contact admin' }`
```

Comment:
we should have 409 as well if the role trying to be updated already pre-exist.

Reply:
Done can you please check in latest commit